docs/kafka/metrics-monitoring-kafka/README.adoc

```diff
@@ -87,76 +87,39 @@ As a developer or administrator, you can view metrics in {product-kafka} to visu
 == Supported metrics in {product-kafka}
 
 [role="_abstract"]
-{product-kafka} supports the following metrics for Kafka instances and topics. In the {product-kafka} web console, a subset of these metrics is displayed in the *Dashboard* page of each Kafka instance.
+{product-kafka} supports the following metrics for Kafka instances and topics. In the {product-kafka} web console, a subset of these metrics is displayed in the *Dashboard* page of each Kafka instance. To learn more about the limits associated with both trial and production Kafka instances, refer to https://access.redhat.com/articles/5979061[Red Hat OpenShift Streams for Apache Kafka Service Limits].
+
 
 Cluster metrics::
 +
 --
-* `kafka_namespace:haproxy_server_bytes_in_total:rate5m`: Number of incoming bytes per second for the cluster in the last five minutes. This metric represents all the data that producers are sending to topics in the cluster.
-+
-The Kafka instance type determines the maximum incoming byte rate, as shown in the following values:
+* `kafka_namespace:haproxy_server_bytes_in_total:rate5m`: Number of incoming bytes per second for the cluster in the last five minutes. This ingress metric represents all the data that producers are sending to topics in the cluster.
+The Kafka instance type determines the maximum incoming byte rate.
 
-** 1 MiB per second for a trial instance
-** 50 MiB per second for a production instance with a size of 1 streaming unit
-** 100 MiB per second for a production instance with a size of 2 streaming units
-
-* `kafka_namespace:haproxy_server_bytes_out_total:rate5m`: Number of outgoing bytes per second for the cluster in the last five minutes. This metric represents all the data that producers are receiving from topics in the cluster.
-+
-The Kafka instance type determines the maximum outgoing byte rate, as shown in the following values:
-
-** 1 MiB per second for a trial instance
-** 100 MiB per second for a production instance with a size of 1 streaming unit
-** 200 MiB per second for a production instance with a size of 2 streaming units
+* `kafka_namespace:haproxy_server_bytes_out_total:rate5m`: Number of outgoing bytes per second for the cluster in the last five minutes. This egress metric represents all the data that producers are receiving from topics in the cluster.
```

Comment:
Think this should be "the data that **consumers** are receiving"?

Reply:
Yes- great catch!